### PR TITLE
Column template - header

### PR DESCRIPTION
**Pull request by zeftilldeath**
A template (only applied to special "series/columns"). To give them visual differentiation and highlight the series they belong to. (The whole panel is clickable).

This PR only addresses the head of the article, there will be further minor changes to the body elements.

As you can imagine this hasn't been built the way I'd like to. But a lot of styles need to be overridden - and I've tried to do this in as clean a way as possible. 

AMP remains un-affected and I've checked this with browsers that don't support css grid/flex and made sure fallbacks are acceptable. 

screen shot 2018-03-01 at 15 55 42

screen shot 2018-03-01 at 15 57 47

screen shot 2018-03-01 at 16 00 14

screen shot 2018-03-01 at 16 00 47

example

### CODE REVIEW

File: common/app/views/fragments/meta/metaInline.scala.html [hunk 1]
```diff
@@ -39,13 +40,27 @@
...
+        <div class="@RenderClasses(Map(
+                "content__series-label--immersive-article" -> (item.content.isImmersive && item.content.tags.isArticle),
+                "content__series-label--photo-essay" -> item.content.isPhotoEssay,
+                "content__series-label--column" -> item.content.isColumn
+            ), "content__series-label")
+          ">
+            <a class="content__series-label__link" href="@LinkTo {/@series.id}">@series.name</a>
+        </div>
+
+        @if((item.content.isColumn && !amp)) {
```
Comment (GHaberis) [inline common/app/views/fragments/meta/metaInline.scala.html]: There's no need to wrap the conditional here in nested brackets

**Review by GHaberis — APPROVED** [review]
some minor comments, but no blockers... 👍

File: article/app/views/fragments/articleBodyGarnett.scala.html [hunk 2]
```diff
@@ -52,9 +53,12 @@
 
 
                         @if(!isPaidContent) {
-                            @if(article.metadata.designType.nameOrDefault == "comment" ||
+                            @if(!article.content.isColumn &&
+                                article.metadata.designType.nameOrDefault == "comment" ||
                                 article.metadata.designType.nameOrDefault == "guardianview") {
                                 @fragments.articleHeaderCommentGarnett(article, model, amp = amp)
+                            } else if(article.content.isColumn) {
+                                @fragments.articleHeaderColumn(article, model, amp = amp)
```
Comment (SiAdcock) [inline article/app/views/fragments/articleBodyGarnett.scala.html]: If we move this branch to the top, we wouldn't need to check for `!article.content.isColumn` in the `articleHeaderCommentGarnett` branch

File: article/app/views/fragments/articleBodyGarnett.scala.html [hunk 3]
```diff
@@ -52,9 +53,12 @@
 
 
                         @if(!isPaidContent) {
-                            @if(article.metadata.designType.nameOrDefault == "comment" ||
+                            @if(!article.content.isColumn &&
```
[the remark this answers is not in the document]
Comment (zeftilldeath) [reply article/app/views/fragments/articleBodyGarnett.scala.html]: 🎉

**Pull request merged**